storage/innobase/xtrabackup/doc/source/using_xtrabackup/privileges.rst

```diff
@@ -68,7 +70,7 @@ alteration, see :option:`mysql --help` for details.
 
    In case of multiple server instances the correct connection parameters
    (port, socket, host) must be specified in order for |xtrabackup| to talk to
-   the correct server. 
+   the correct server.
 
 
 Permissions and Privileges Needed
```

Comment:
Should this page also mention PROCESS privilege for 2.4 since this PR fixing https://bugs.launchpad.net/percona-xtrabackup/+bug/1594990 ?